### ISSUE TITLE
[Feature Request] Toggle Ridden Entity Visibility

### ISSUE THREAD

**Issue by BrandonItaly** [tr7zw/PaperDoll#22]
Allow the visibility of ridden entities to be toggled to match the behavior of Bedrock Edition.

**Comment by tr7zw** [comment]
You mean, you want to be able to turn that off? Also as a note(to the other issue too), this mod doesn't aim to have parity/resemblence/reproduce bedrock/pocket edition. I never played bedrock(other than short messing around on the win10 edition to decide that I don't like it) or the pocket editions.

**Comment by BrandonItaly** [comment]
Yeah, I'd like to be able to turn it off. No worries if that's something you don't want to implement though

**Comment by tr7zw** [comment]
Turning off stuff is usually simpler than adding 😅. I will try to add it in the next release, probably being 1.20.3(as I already kinda forgot to make 1.20.2 due to irl stress).
